llvm/include/llvm/Frontend/HLSL/HLSLResource.h

```diff
@@ -25,6 +25,7 @@ enum class ResourceClass : uint8_t {
   SRV = 0,
   UAV,
   CBuffer,
+  TBuffer,
```

Comment:
TBuffer should be a resource kind, not resource class.

Reply:
I hadn't noticed this. Xiang's right, this isn't correct. See `CGHLSLRuntime::finishCodeGen` - a tbuffer is annotated with a resource class of `SRV` as you would expect.